Python/Color_Picker/README.md

```diff
@@ -0,0 +1,27 @@
+# COLOR_PICKER FROM IMAGE
+
+Picking color from image is used in the computer Vision application.Here is the python script which gives the name of the color and it's RGB value when clicked on the image. These scripts uses OpenCV, pandas and argpass.
+Before running the script make sure you have OpenCV installed in system.
+
+# Set Up
+Before running script make sure your system has Open Cv and pandas.
+
+
+If not then do download it using```"pip install opencv-python"``` and ```"pip install pandas"```
```

Comment:
It would be great if u can upload a `requirements.txt` put the dependencies in it.